scan/lib/scan/detect_values.rb

```diff
@@ -175,10 +175,26 @@ def self.detect_destination
 
       # building up the destination now
       if Scan.devices && Scan.devices.count > 0
```

Comment:
Ah yea this seems to have been an error while rebasing the branch, i wasn't aware of that issue. Happy to open a pull request with a fix for this.